bcda/client/bluebutton.go

```diff
@@ -256,3 +259,13 @@ func HashIdentifier(toHash string) (hashedValue string) {
 	}
 	return hex.EncodeToString(pbkdf2.Key([]byte(toHash), pepper, blueButtonIter, 32, sha256.New))
 }
+
+func UpdateParamWithLastUpdated(params *url.Values, since string, jobCreation time.Time) {
+	// only set the lower bound parameter if it exists and begins with "ge" (to align with what is expected in _lastUpdated)
+	if len(since) > 0 && strings.HasPrefix(since, "ge") {
+		params.Set("_lastUpdated", since)
+
```

Comment:
I'm suspicious that this part of the code is using a string instead of the time type. It looks this string comes from the query parameter of the request if I read the code correctly. If so, it seems like this code would allow SQL injection attacks on the BFD.

Reply:
Input validation is done here: https://github.com/CMSgov/bcda-app/pull/459/files#diff-9b652353711032572a163389de40e045R263